tezos/messages/src/p2p/encoding/peer.rs

```diff
@@ -39,7 +39,7 @@ pub enum PeerMessage {
     OperationsForBlocks(OperationsForBlocksMessage),
 }
 
-#[derive(Serialize, Deserialize, Debug, Getters)]
+#[derive(Serialize, Deserialize, Debug, Getters, Clone)]
```

Comment:
@akoptelov 
where do we need to clone PeerMessageResponse now?

Reply:
I thought it was needed for write test, but obviously it is not now.